backend/src/personnel/personnel.service.ts

```diff
@@ -130,14 +149,15 @@ export class PersonnelService {
     const bcws = person.bcws;
     const emcr = person.emcr;
 
-    if (personnel.tools?.[0]?.hasOwnProperty('tool')) {
-      const allTools = await this.toolsRepository.find();
-      const personnelTools = personnel.tools.map((t) => ({
-        toolId: allTools.find((at) => at.name === t.tool).id,
-        proficiencyLevel: t.proficiencyLevel,
-      }));
-      personnel.tools = personnelTools;
-    } else {
+    if (!personnel.tools?.[0]?.hasOwnProperty('tool')) {
+    //   console.log(personnel.tools)
```

Comment:
This meant to be commented? I had thought this was working

Reply:
Sorry I will revert this - was having some issues updating re: tool Id.